_gsocorgs/2020/slac.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "SLAC"
+author: "Andrew Hanushevsky"
+layout: default
+organization: SLAC
+logo: SLAC-logo.png
+description: | 
+  [SLAC National Accelerator Laboratory](www6.slac.stanford.edu), originally named Stanford Linear Accelerator Center is a United States Department of Energy National Laboratory operated by Stanford University under the programmatic direction of the U.S. Department of Energy Office of Science and located in Menlo Park, California. It is the site of the Stanford Linear Accelerator, a 3.2 kilometer (2 mile) linear accelerator constructed in 1966 and shut down in the 2000s, which could accelerate electrons to energies of 50 GeV. Today SLAC research centers on a broad program in atomic and solid-state physics, chemistry, biology, and medicine using X-rays from synchrotron radiation and a free-electron laser as well as experimental and theoretical research in elementary particle physics, astroparticle physics, and cosmology.
```

Comment:
the link here has to be explicit, i.e. https://www6.slac.stanford.edu/ otherwise it is considered as a local server link

Reply:
Done